Makefile

```diff
@@ -1,5 +1,10 @@
+UNVERSIONED = unversioned
+
 artifact_name       := test-data-generator
-version             := "unversioned"
+version             := ${UNVERSIONED}
+docker_registry     := 169942020521.dkr.ecr.eu-west-2.amazonaws.com
```

Comment:
We shouldn't be hardcoding registry urls. I'd be passing this in

Reply:
Updated to...
_`docker_registry     := ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com`_
...which should make the format and requirements clear. Added in caps to indicate this is an env var.